README.md

```diff
@@ -1,18 +1,22 @@
 # STAC Project Steering Committee
 
-This document describes how the STAC (SpatioTemporal Asset Catalog) Project Steering Committee makes decisions on STAC and STAC API project issues. STAC is concerned with both the [stac-spec](https://github.com/radiantearth/stac-spec) and [stac-api-spec](https://github.com/radiantearth/stac-api-spec) repositories.
+This document describes how the STAC (SpatioTemporal Asset Catalog) Project Steering Committee makes decisions on STAC and STAC API project issues. STAC is concerned with the following GitHub repositories and organizations:
+- [stac-spec](https://github.com/radiantearth/stac-spec)
+- [stac-api-spec](https://github.com/radiantearth/stac-api-spec)
+- [stac-extensions](https://github.com/stac-extensions)
+- [stac-api-extensions](https://github.com/stac-api-extensions)
```

Comment:
I think it makes sense to also include the extensions as there are various core extensions in there and we did the maturity evaluations in the past.